lametro/templates/search/_ordered_search_filter_items.html

```diff
@@ -5,11 +5,14 @@
     {% if count %}
         {% if name in selected_list %}
             <li class="small">
-                <a href="#" class="filter-value" data="{{facet_name}}_exact:{{name}}" title="{{ name }}"><strong>{{ name | short_topic_name }}</strong></a>
-                <a href ="#" class="remove-filter-value" data="{{facet_name}}_exact:{{name}}">
-                   <i class="fa fa-times"></i>
+                <a href="#" class="filter-value" data="{{facet_name}}_exact:{{name}}" title="{{ name }}">
+                    <strong>{{ name | short_topic_name }}</strong>
                 </a>
-                <span class="badge badge-facet pull-right">{{ count }}</span>
```

Comment:
The "Filter by Meeting" accordion seems to be missing the count.

<img width="462" alt="Screen Shot 2023-06-28 at 4 03 58 PM" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/36973363/c0c225ef-9285-4f4e-b4c8-ed53bed524b4">

Reply:
Ah you're right. There is a conditional in [`_search_filter.html`](lametro/templates/search/_search_filter.html) specifically for sponsorships (the Filter by Meeting section) that I carried over:

[`{% if facets.fields.legislative_session|length > 1 %}` ](https://github.com/Metro-Records/la-metro-councilmatic/pull/988/files/064d119fb3746964906d62104f00c4f8859465ac#diff-fc097a686fb9405f2f370e47babde9e2512d8b8ce3544fe02cc35daf7d7afdcdL32)

that was preventing counts from rendering for those sponsorships. Looking at what it's used for, it only changes whether or not to leave space for and render the count.

In a new commit I got rid of that conditional and now space is always allotted for the count. So it gets rendered if it exists and just leaves a bit of space if it doesn't. I figure it would help keep facet name lengths uniform. What do you think?

Example of if only some have a count:
![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/7017952d-f200-4cec-b993-e764520fd625)